server/routerlicious/packages/lambdas/src/alfred/index.ts

```diff
@@ -683,6 +692,11 @@ export function configureWebSocketServices(
             // Send notification messages for all client IDs in the room map
             for (const [clientId, room] of roomMap) {
                 const messageMetaData = getMessageMetadata(room.documentId, room.tenantId);
+                // excluding summarizer for total client count.
+                if (connectionTimeMap.has(clientId)) {
+                    // eslint-disable-next-line @typescript-eslint/no-floating-promises
+                    connectionCountLogger.logConnectionCount(false);
```

Comment:
would
`void connectionCountLogger.logConnectionCount(false);`
work? That might allow to remove the eslint disable rule

Reply:
the disable rule is because we are not awaiting on the call. Required because we dont want to block the processing of the request for logging this metric.

Reply:
I think `void connectionCountLogger.logConnectionCount(false);` achieves the same. I think many of these fire-and-forget scenarios have been using void throughout the repo. See https://github.com/microsoft/FluidFramework/blob/b1d44311395879fdd65f097efe8ba849147e5f94/server/routerlicious/packages/lambdas/src/deli/checkpointContext.ts#L89 for an example.

Reply:
Ah! I see what you're saying. Actually I see that we are using both approaches in r11s, with void and with eslint rule. Personal preference I guess?

Reply:
Oh! I didn't realize. I think I had to deal with this scenario once only, and when I looked through the code, I found that `void` strategy and used it. I did a quick research and it seems there is no difference between the 2 options (`void` vs. eslint disable). So feel free to use whichever one you prefer 😄